ci/build/release-prep.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Description: This is a script to make the release process easier
+# Run it with `yarn release:prep` and it will do the following:
+# 1. Check that you have a $GITHUB_TOKEN set and hub installed
+# 2. Update the version of code-server (package.json, docs, etc.)
+# 3. Update the code coverage badge in the README
+# 4. Open a draft PR using the release_template.md and view in browser
+# If you want to perform a dry run of this script run DRY_RUN=0 yarn release:prep
+
+set -euo pipefail
+
+main() {
+  if [ "${DRY_RUN-1}" -eq 0 ]; then
```

Comment:
Thoughts on comparing to one instead? Setting `DRY_RUN` to zero feels to me like it should be falsy. 

We also might want to use `=` in case `DRY_RUN` is blank or some other non-integer.

```suggestion
  if [ "${DRY_RUN-}" = 1 ]; then
```

Reply:
Doesn't matter to me - I guess `0` is falsey in the JS world.

Makes sense though and good call! I'll fix. 

(note: I probably still need a number after the `-` in `${DRY_RUN-}` right? otherwise it will be undefined and exit the script?)

Reply:
Actually, I see your comment below. TIL!